src/HealthCheck.Core/Results/CheckResult.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace HealthCheck.Core.Results
+{
+    public class CheckResult : ICheckResult
+    {
+        public string CheckerName { get; set; }
```

Comment:
Renaming `Name` to `CheckerName` is a pretty rough breaking change. I can see that `CheckerName` could be a better name, but I'm not sure it's _better enough_ to justify breaking all real-world tests.

Reply:
Well, i,d keep it that way. Breaking changes are not avoidable. Checker has SessionName already. Since we have multiple names in one class lets name it as strict as possible